### PR TITLE
Remove Flask-API extension dependency

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
tendrl-bug-id: Tendrl/performance-monitoring#31
Signed-off-by: Timothy Asir J <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@r0h4n , @nthomas-redhat , @shtripat , @anmolbabu , Please review

File: tendrl/performance_monitoring/api/manager.py [hunk 1]
```diff
@@ -14,6 +13,7 @@
 persister = None
 time_series_db_manager = None
 app = Flask(__name__)
+HTTP_500_INTERNAL_SERVER_ERROR = 1
```
Comment (shtripat) [inline tendrl/performance_monitoring/api/manager.py]: @anmolbabu is this return code fine??

Comment (anmolbabu) [inline tendrl/performance_monitoring/api/manager.py]: 500 is supposed to be the code

Comment (TimothyAsirJeyasing) [inline tendrl/performance_monitoring/api/manager.py]: Done

**Comment by TimothyAsirJeyasing** [comment]
@anmolbabu , @shtripat  , Please review

**Comment by TimothyAsirJeyasing** [comment]
Its now part of https://github.com/Tendrl/performance-monitoring/pull/38. I am closing this patch

**Pull request closed without merging**